src/Microsoft.Data.SqlClient/tests/Directory.Build.props

```diff
@@ -15,20 +16,20 @@
     <!-- Set Default Target Framework when building for Debug and Release configurations. (Visual Studio) -->
     <When Condition="'$(TestTargetOS)' == '' AND ('$(Configuration)' == 'Debug' OR '$(Configuration)' == 'Release')">
       <PropertyGroup>
-        <TargetFramework>$(TargetNetCoreVersion)</TargetFramework>
+        <TargetFrameworks>$(TargetNetCoreVersion)</TargetFrameworks>
```

Comment:
This property has to be changed because the `TargetFramework` property in NSLibrary.csproj was also changed to `TargetFrameworks`. When this .props file is imported into NSLibrary.csproj during a build, the `TargetFrameworks` property in NSLibrary.csproj will overwrite this property.